lmfdb/belyi/main.py

```diff
@@ -908,3 +969,77 @@ def __init__(self):
         self.browse_array = [[deg], [group], [abc], [abc_list], [g], [orbit_size], [pass_size], [field], [geomtype], [is_primitive], [primitivization], [count]]
 
         self.refine_array = [[deg, group, abc, abc_list], [g, orbit_size, pass_size, field], [geomtype, is_primitive, primitivization]]
+
+class PassportSearchArray(SearchArray):
```

Comment:
This is also almost identical to the search array for Belyi maps.  It should be possible to use inheritance to factor out a bunch of common code.

Reply:
Sounds good. I'll revise this PR this week.